lotus/client.go

```diff
@@ -58,11 +59,27 @@ func SetupClientFromCLI(cctx *cli.Context) (tasks.NodeConfig, api.FullNode, Node
 	if err != nil {
 		return tasks.NodeConfig{}, nil, nil, fmt.Errorf("wallet is not a Filecoin address: %s, %s", cctx.String("wallet"), err)
 	}
+	mf := big.NewInt(100000000000000)
```

Comment:
This number `100000000000000` is used in multiple places. Suggest using a `const`.